components/api-gateway-migrator/README.md

```diff
@@ -0,0 +1,86 @@
+## Overview
+
+This program translates old api objects (Api) to a new one (ApiRule)
+Usage:
+`go run cmd/main.go --label-blacklist=migration/status`
+
+To see all possible arguments, use the following:
+`go run cmd/main.go -h`
+
+After migration, old Api objects are disabled - the host is randomized to point to a non-existing location.
+Objects themselves are not deleted. Users can delete all migrated Api objects with the following command:
+`kubectl delete apis -l migration/status=migrated --all-namespaces`
```

Comment:
I would change it to the following: 
(I haven't use the suggestion here since it didn't work well with several lines + code)

## Usage

To migrate APIs to APIRules, run: 

```bash
go run cmd/main.go --label-blacklist=migration/status
```
To see all possible arguments, run: 
```bash
go run cmd/main.go -h
```

After migration, the host is randomized to point to a non-existing location. This disables the existing APIs, but does not remove the objects. To delete all migrated API objects, run:
```bash
kubectl delete apis -l migration/status=migrated --all-namespaces
```

Reply:
Done.